go.sum

```diff
@@ -281,6 +283,8 @@ github.com/fatih/structtag v1.1.0 h1:6j4mUV/ES2duvnAzKMFkN6/A5mCaNYPD3xfbAkLLOF8
 github.com/fatih/structtag v1.1.0/go.mod h1:mBJUNpUnHmRKrKlQQlmCrh5PuhftFbNv8Ys4/aAZl94=
 github.com/flynn/go-shlex v0.0.0-20150515145356-3f9db97f8568/go.mod h1:xEzjJPgXI435gkrCt3MPfRiAkVrwSbHsst4LCFVfpJc=
 github.com/fortytw2/leaktest v1.3.0/go.mod h1:jDsjWgpAGjm2CA7WthBh/CdZYEPF31XHquHwclZch5g=
+github.com/fromanirh/pack8s v0.1.0 h1:n7okFfusL81MTmtlVhWFZcss8JtMxi1HeOwUe6SkJkk=
```

Comment:
Is this the same as the pack8s kept under kubevirt/kubevirtci https://github.com/kubevirt/kubevirtci/tree/master/pack8s?

Reply:
This is indirect dependency. Was updated by go mod tidy.